### ISSUE TITLE
Voiceover reads dropdown items twice

### ISSUE THREAD

**Issue by gptt916** [carbon-design-system/carbon#7707]
## Environment

> Operating system

Mac osx 11.1

> Browser

Safari

> Automated testing tool and ruleset

https://www.ibm.com/able/guidelines/ci162/info_and_relationships.html

> Assistive technology used to verify

Mac voice over (command f5), then move focus from one item to the next, items are read twice.

## Detailed description

> What version of the Carbon Design System are you using?

latest

> What did you expect to happen?

voice over read item once

> What happened instead?

Items are read twice

> What WCAG 2.1 checkpoint does the issue violate?

https://www.ibm.com/able/guidelines/ci162/info_and_relationships.html

## Steps to reproduce the issue

1. Go to https://carbon-components-react.netlify.app/?path=/story/dropdown--default
2. On a mac, press command f5
3. Open the dropdown, then using the keyboard, navigate the focus from one item to the next
4. Voice over reads items twice

## Additional information

- Screenshots or code
- Notes
![image](https://user-images.githubusercontent.com/20601623/106483304-5f264d00-647c-11eb-9cf0-424744e4ea44.png)

**Comment by aledavila** [comment]
Hi @gptt916 thanks for submitting this. I’m going to close this as it’s default behavior. In this instance, the title and the description in the element but that’s not always the case. VO is saying both as it should be.

**Comment by carmacleod** [comment]
@aledavila Actually, could you reopen please?
I think the items should only have a title tooltip when there's an ellipsis.
i.e. only the horribly long item should be read twice (and I even think there's got to be a way to work around that 😂 ).
(Also, I think the horribly long item is way too long, and so is setting a bad example. Nobody should have a dropdown item _that_ long. If possible, please delete everything after "elit". It will still show what you want to show.)
